include/circt/Dialect/SV/SVTypeDecl.td

```diff
@@ -206,6 +208,50 @@ def ModportType : SVType<"Modport"> {
   let assemblyFormat = "`<` $modport `>`";
 }
 
+//===----------------------------------------------------------------------===//
+// Verbatim module operation
+//===----------------------------------------------------------------------===//
+
+def SVVerbatimModuleOp
+    : SVOp<"verbatim.module", [DeclareOpInterfaceMethods<PortList>,
+                               DeclareOpInterfaceMethods<HWModuleLike>, Symbol,
+                               HasParent<"mlir::ModuleOp">]> {
+  let summary = "Module with verbatim verilog content";
+  let description = [{
+    The `sv.verbatim.module` operation represents a module with verbatim content.
+    It is a `HWModuleLike` (like `hw.module` and `hw.module.extern`).
+    To support verbatim content split accross multiple files, there is an optional
+    additional_files field which can reference additional emitted files.
+  }];
+
+  let arguments = (ins SymbolNameAttr:$sym_name,
+      TypeAttrOf<ModuleType>:$module_type, StrAttr:$content,
+      OutputFileAttr:$output_file, OptionalAttr<DictArrayAttr>:$per_port_attrs,
+      OptionalAttr<LocationArrayAttr>:$port_locs,
+      ParamDeclArrayAttr:$parameters,
+      OptionalAttr<SymbolRefArrayAttr>:$additional_files);
```

Comment:
Please add SymbolUserOpInterface to SVVerbatimModuleOp since additional_files refer to symbols.